velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -456,5 +456,49 @@ TEST_F(DateTimeFunctionsTest, quarterDate) {
   EXPECT_EQ(3, quarter("1954-08-08"));
 }
 
+TEST_F(DateTimeFunctionsTest, fromUnixTime) {
+  const auto getUnixTime = [&](const StringView str) {
+    Timestamp t = util::fromTimestampString(str);
+    return t.getSeconds();
+  };
+
+  const auto fromUnixTime = [&](std::optional<int64_t> unixTime,
+                                std::optional<std::string> timeFormat) {
```

Comment:
Why do we need `unixTime` and `timeFormat` to be optional?

Reply:
It looks evaluate requires optional input.